src/module-elasticsuite-swatches/Block/Navigation/Renderer/Swatches/RenderLayered.php

```diff
@@ -26,6 +27,50 @@
  */
 class RenderLayered extends \Magento\Swatches\Block\LayeredNavigation\RenderLayered
 {
+    /**
+     * @return array
+     */
+    public function getSwatchData(): array
+    {
+        if (false === $this->eavAttribute instanceof Attribute) {
+            throw new \RuntimeException('Magento_Swatches: RenderLayered: Attribute has not been set.');
+        }
+
+        $attributeOptions = [];
+        $attributeOptionsSort = [];
+
+        $sortingArr = [];
+        foreach ($this->filter->getItems() as $item) {
+            $sortingArr[] = $item['label'];
+        }
+
```

Comment:
2) Explain that you are here building an array whose keys are the attribute option label and not option id as in the native method.